tests/unit/filing-template-mixin.spec.ts

```diff
@@ -193,7 +193,7 @@ describe('Registration Filing', () => {
         naicsCode: '12345',
         naicsDescription: 'Some NAICS Description'
       },
-      businessTypeConfirm: false,
+      businessTypeConfirm: false
```

Comment:
Another lint fix.

(It's weird that lint allows a trailing comma for enums but not objects but oh well.)